server/entity/MediaRequest.ts

```diff
@@ -1187,3 +1187,5 @@ export class MediaRequest {
     }
   }
 }
+
+export default MediaRequest;
```

Comment:
Why did we add this default export?

Reply:
To keep consistency with the other entities that had default exports.